Resources/Prototypes/Voice/speech_emotes.yml

```
@@ -2,7 +2,7 @@
 - type: emote
   id: Scream
   category: Vocal
-  chatMessages: [screams, yells]
+  chatMessages: [кричит, орёт]
```

Отметь изменения строками
```suggestion
  chatMessages: [кричит, орёт] # Corvax-Localization
```